Know Your Code/C++/Radix Sort

```diff
@@ -0,0 +1,74 @@
+#include <iostream>
+#include <vector>
+
+using namespace std;
+
+// Function to find the maximum number to determine the number of digits
+int getMax(vector<int>& arr) {
+    int max = arr[0];
+    for (int i = 1; i < arr.size(); i++) {
+        if (arr[i] > max) {
+            max = arr[i];
+        }
+    }
+    return max;
+}
+
+// Using counting sort to sort the elements based on significant places
+void countingSort(vector<int>& arr, int exp) {
```

Comment:
void countingSort(vector<int>& arr, const int exp) {

Reply:
Use const for read-only variables

Reply:
Okay